src/org-manager/services/organisation.service.ts

```diff
@@ -1,13 +1,15 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { Observable, of } from 'rxjs';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Observable, of, throwError } from 'rxjs';
 import { OrganisationsMock } from '../mock/organisation.mock';
 import { Organisation } from '../models/organisation';
 import {SingleOrgSummary} from '../models/single-org-summary';
 import {SingleOrgSummaryMock} from '../mock/single-org-summary.mock';
+import { tap, catchError } from 'rxjs/operators';
 
 @Injectable()
 export class OrganisationService {
+  private singleOrgUrl = 'http://localhost:3000/organisations?pbaNumber=';
```

Comment:
this should live in configuration file... maybe evnironment.ts

Reply:
done